.github/workflows/build-test.yml

```
@@ -18,7 +18,7 @@ jobs:
     uses: playframework/.github/.github/workflows/cmd.yml@v3
     with:
       java: 17, 11, 8
-      scala: 2.12.20
+      scala: 2.12.20, 3.3.4
```

```suggestion
      scala: 2.12.20, 3.7.2
```
To match the version that got merged with #279